### ISSUE TITLE
[Question] Frontend compilation failed with dependency issue with grunt-connect-proxy

### ISSUE THREAD

**Issue by peacand** [TheHive-Project/TheHive#2380]
### Request Type

Question : frontend build from sources

### Work Environment

| Question                   | Answer                                |
| -------------------------- | ------------------------------------- |
| OS version (server)        | Debian   |
| OS version (client)        | Windows 10 |
| Virtualized  Env.          | True / False                          |
| Dedicated RAM              | 8 GB                                 |
| vCPU                       | 2                     |
| TheHive version / git hash | 4.1.19               |
| Package Type               | From source |
| Database                   | BerlkelyDB                |
| Index type                 | Lucene                 |
| Attachments storage        | Local                  |

### Question

Hi,

I'm trying to build theHive 4.1.19. The frontend compilation failed with a dependency error related to **grunt-connect-proxy**.

Basically : 

- thehive module depends on : grunt-connect-proxy@0.2.0 + grunt@1.1.0
- grunt-connect-proxy@0.2.0 depends on grunt@0.4.1

So there is a broken dependency here. I've managed to fix it by changing the dependency with grunt-connect-proxy2. Which is a fork but without the hard dependency with grunt@0.4.1. This one is working with grunt newer versions. 

Did I miss something ? Could someone explain me the official way to build theHive4 without having this dependency issue ?

Thank you for your help

**Comment by priamai** [comment]
Plus one of this thanks for the fix I never managed to compile it!
